WORKSPACE

```diff
@@ -1,5 +1,31 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+http_archive(
+   name = "rules_foreign_cc",
+   strip_prefix = "rules_foreign_cc-master",
+   url = "https://github.com/bazelbuild/rules_foreign_cc/archive/master.zip",
+)
+
+load("@rules_foreign_cc//:workspace_definitions.bzl", "rules_foreign_cc_dependencies")
+
+rules_foreign_cc_dependencies()
+
+http_archive(
+    name = "libelf",
+    url = "https://fossies.org/linux/misc/old/libelf-0.8.13.tar.gz",
```

Comment:
uhm .. the url looks suspicious (old/, and fossies.org). Is this the correct libelf? By looking at the ChangeLog, it was last updated in 2009. The README says it can be downloaded from http://www.mr511.de/software/, but the site is down. How did you find it?

Debian uses the libelf in the elfutils package, from here: https://sourceware.org/elfutils/, so does Ubuntu.

Reply:
I did verify that it was the latest version but I had used it for test purposes when i was experimenting. Will update to pick from elfutils and open in a separate PR

Reply:
https://sourceware.org/elfutils/ftp/0.177/elfutils-0.177.tar.bz2

Reply:
Thanks Marcus.